src/components/NftsPanel.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <app-panel>
+    <paginated-content
+      v-model:page-index="pageIndex"
+      :entities="nfts"
+      :total-count="nftsCount"
+      :limit="limit"
+      pagination-style="history"
+      @prev-clicked="loadPrevNfts"
+      @next-clicked="loadNextNfts">
+      <nfts-table
+        :nfts="nfts"
+        class="nfts-panel__table"/>
+      <nfts-table-condensed
+        :nfts="nfts"
+        class="nfts-panel__table-condensed"/>
+    </paginated-content>
+  </app-panel>
+</template>
+
+<script setup>
+import { storeToRefs } from 'pinia'
+import { useNftsStore } from '@/stores/nfts'
+import { isDesktop } from '@/utils/screen'
+
+const limit = computed(() => process.client && isDesktop() ? 10 : 3)
+const pageIndex = ref(1)
+
+const nftsStore = useNftsStore()
+const {
+  nfts,
+  nftsCount,
+} = storeToRefs(nftsStore)
+const { fetchNfts, fetchNftsCount } = nftsStore
+
+await fetchNftsCount()
+await fetchNfts()
```

Comment:
looks like limit is unused during fetching. This should be fetched on client side

Reply:
fixed